src/main/java/com/kryptnostic/rhizome/emails/EmailStreamSerializer.java

```diff
@@ -0,0 +1,125 @@
+package com.kryptnostic.rhizome.emails;
+
+import java.io.IOException;
+import java.util.Date;
+import java.util.List;
+
+import jersey.repackaged.com.google.common.collect.Lists;
+import jodd.mail.Email;
+import jodd.mail.EmailMessage;
+import jodd.mail.MailAddress;
+
+import com.geekbeast.rhizome.pods.hazelcast.SelfRegisteringStreamSerializer;
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+
+public class EmailStreamSerializer implements SelfRegisteringStreamSerializer<Email> {
+
+    @Override
+    public void write( ObjectDataOutput out, Email object ) throws IOException {
+        serialize( out, object );
+    }
+
+    @Override
+    public Email read( ObjectDataInput in ) throws IOException {
+        // TODO Auto-generated method stub
+        return null;
```

Comment:
is there a specific reason for returning `null`? can you return an empty `Email` object instead?